spark-atlas-connector/src/main/scala/com/hortonworks/spark/atlas/sql/CommandsHarvester.scala

```diff
@@ -95,22 +111,40 @@ object CommandsHarvester extends AtlasEntityUtils with Logging {
             case r: FileRelation => r.inputFiles.map(external.pathToEntity).toSeq
             case _ => Seq.empty
           }
+        case local: LocalRelation =>
+          logInfo("Local Relation")
+          Seq.empty
         case e =>
           logWarn(s"Missing unknown leaf node: $e")
           Seq.empty
       }
 
+      val inputTablesEntities = if (isFiles) inputsEntities.flatten.toList
+      else inputsEntities.flatMap(_.headOption).toList
+
       // new table/file entity
       val outputEntities = node.catalogTable.map(tableToEntities(_)).getOrElse(
         List(external.pathToEntity(node.outputPath.toUri.toString)))
 
-      // create process entity
-      val inputTablesEntities = if (isFiles) inputsEntities.flatten.toList
-        else inputsEntities.flatMap(_.headOption).toList
-      val outputTableEntities = List(outputEntities.head)
-      val pEntity = processToEntity(qd.qe, qd.executionId, qd.executionTime, inputTablesEntities,
-        outputTableEntities, qd.query)
-      Seq(pEntity) ++ inputsEntities.flatten ++ outputEntities
+      val logMap = Map(
+        "executionId" -> qd.executionId.toString,
+        "remoteUser" -> SparkUtils.currSessionUser(qd.qe),
+        "executionTime" -> qd.executionTime.toString,
+        "details" -> qd.qe.toString(),
+        "sparkPlanDescription" -> qd.qe.sparkPlan.toString())
+
+      // ml related cached object
+      if (internal.cachedObjects.contains("model_uid")) {
+
+        internal.updateMLProcessToEntity(inputTablesEntities, outputEntities, logMap)
+
```

Comment:
Sorry for nit-picking, but let's not put blank space like this line 138 and line 140. There are more occurences like this. :)